pipelines/datasets/world_wb_wdi/constants.py

```
@@ -0,0 +1,63 @@
+    SOURCE_URL = "https://databank.worldbank.org/data/download/WDI_CSV.zip"
+    ZIP_NAME = "WDI_CSV.zip"
+
+    # Source file (inside the zip) per output table. ``data`` and ``dicionario``
+    # both derive from WDICSV.csv / WDISeries.csv and are handled specially.
+    SOURCE_FILES = {
+        "data": "WDICSV.csv",
+        "indicators": "WDISeries.csv",
+        "country_indicator": "WDIcountry-series.csv",
+        "footnote": "WDIfootnote.csv",
+        "indicator_time": "WDIseries-time.csv",
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)